tests/test_downloader.py

```diff
@@ -89,6 +89,33 @@ def test_download_single_song(setup):
         "test artist - test song.mp3"
     ]
 
+@pytest.mark.vcr()
+def test_download_long_artists_song(setup):
+    # ! Generates a long list of artists, numbered 1 to 260, to trigger filename length cases
+    artists = list(map(lambda x:str(x),range(260)))
```

Comment:
This produces exactly the same output but is less complicated.
```suggestion
    artists = [i for i in range(260)]
```